website/blog/2023-05-26-7.22.0.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title:  "7.22.0 Released: Explicit Resource Management support and Import Attributes parsing"
+author: Babel Team
+authorURL: https://twitter.com/babeljs
+date:   2022-05-26 0:00:00
+categories: announcements
+share_text: "Babel 7.22.0 Released"
+---
+
+Babel 7.22.0 is out, with parsing/transform support for the [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management) proposal, including both the sync and async variants, and with parsing support for the [Import Attributes](https://github.com/tc39/proposal-import-attributes) (an evolution of the old [Import Assertions](http://web.archive.org/web/20220930084518/https://github.com/tc39/proposal-import-assertions) proposal).
+
+We also updated our implementation of [decorators](https://github.com/tc39/proposal-decorators/) following some changes in the proposal, and added support for the TypeScript `import ... =` and `export ... =` statements.
+
+`@babel/preset-env` now includes transform support for the `v` regular expressions flag, which was recently approved as part of the ECMAScript standard, by default. Lastly, we renamed all the plugins for _stable_ ECMAScript features from `-proposal-` to `-transform-`.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.22.0).
+
+<!-- truncate -->
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://github.com/babel/babel?sponsor=1) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### Explicit Resource Management ([#15633](https://github.com/babel/babel/pull/15633), [#15520](https://github.com/babel/babel/pull/15520))
+
+The [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management/) Stage 3 proposal allows defining variables containing resources that will be "disposed" when exiting the scope where they are declared. This is done through `using` (for synchronous disposal) and `await using` (for asynchronous disposal) declarations:
+
+```javascript
```

Comment:
Just curious, why do we use both `javascript` and `js title="JavaScript"`?

Reply:
We should always use `js title="JavaScript"` (or `javascript title="JavaScript"`):
- `js` is for the syntax highlighting
- `title="JavaScript"` is to show it as the code block title